cpg_utils_config.md

```diff
@@ -4,6 +4,40 @@ The [`cpg-utils` library](https://github.com/populationgenomics/cpg-utils) ([pyp
 
 This configuration tool uses one or more `TOML` files, and creates a dictionary of key-value attributes which can be accessed at any point, without explicitly passing a configuration object. If jobs are set up using `analysis-runner`, config will be set up automatically within each job environment. Please see the end section of this document for extra details on how to set up config outside analysis-runner.
 
```

Comment:
I think adding some extra information that provides more context for configs and what they're used for would be beneficial. 

`Config files are used in combination with analysis-runner as a way to set parameters for the entirety of the workflow/script we are wanting to run. These parameters determine critical aspects such as the choice of reference genome and the specific stages of the workflow to be executed. They allow us to be flexible in our analyses of data as we can adjust parameters as we progress.`

Reply:
Thanks for the suggestion, I've melded some stuff that was written in the analysis-runner tutorial